src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Telemetry/TelemetryReporter.cs

```diff
@@ -266,6 +265,24 @@ public TelemetryScope TrackLspRequest(string lspMethodName, string languageServe
             new("eventscope.correlationid", correlationId));
     }
 
+    public void UpdateRequestTelemetry(string name, string? language, TimeSpan queuedDuration, TimeSpan requestDuration, AspNetCore.Razor.Telemetry.TelemetryResult result, Exception? exception)
```

Comment:
Currently this isn't using the `Exception?`. Roslyn doesn't keep it, and I'm not sure we could reliably report a histogram with any meaningful thing other than an exception count.